lib/fastly/client.rb

```diff
@@ -152,7 +142,6 @@ def headers(extras={}, include_auth=true)
       headers = {}
       # Some endpoints (POST /tokens) break if any auth headers including cookies are sent
```

Comment:
Now that we're removing support for cookies, then I presume this code comment can be deleted as well.